pkg/apiserver/api_installer.go

```diff
@@ -286,6 +290,9 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	scope := mapping.Scope
 	nameParam := ws.PathParameter("name", "name of the "+kind).DataType("string")
 	pathParam := ws.PathParameter("path", "path to the resource").DataType("string")
+	exportParam := ws.QueryParameter("export", "strip fields for export").DataType("boolean")
+	exactParam := ws.QueryParameter("exact", "if true, maintain cluster-specific fields (e.g. Namespace)").DataType("boolean")
```

Comment:
This helps, thanks.  When I was working with it, I was surprised when `--exact` still changed things before I got the object.